lib/cocoapods/installer/analyzer.rb

```diff
@@ -217,7 +217,7 @@ def generate_podfile_state
       def update_repositories
         sources.each do |source|
           if source.git?
-            config.sources_manager.update(source.name)
+            config.sources_manager.update(source.name, true)
```

Comment:
I was actually thinking that we could start doing this by default for sources/specs. You don't usually know you need it until after you wish you had it 😭